next/components/organisms/NavBar/AlertBanner.tsx

```diff
@@ -0,0 +1,59 @@
+import { AlertIcon, CrossIcon } from '@assets/images'
+import Markdown from '@components/atoms/Markdown'
+import Button from '@components/forms/simple-components/Button'
+import SectionContainer from '@components/ui/SectionContainer/SectionContainer'
+import { useGeneralContext } from '@utils/generalContext'
+import cx from 'classnames'
+import React, { forwardRef, useEffect, useState } from 'react'
+import { useLocalStorage } from 'usehooks-ts'
+
+const AlertBanner = forwardRef<HTMLDivElement>((props, forwardedRef) => {
+  const { alert } = useGeneralContext()
+  const { text, updatedAt } = alert?.data?.attributes ?? {}
+  const storageKey = `bratislava-sk-dismissible-alert-timestamp_${updatedAt}`
```

Comment:
I still think that saving the timestamp as value, not key is better, as this pollutes local storage with old keys, in value case, there would be only one key/value pair.